src/bundle/Resources/views/content/tabs/versions.html.twig

```diff
@@ -0,0 +1,18 @@
+<div class="ez-versions-list-container">
+    {%  if draftVersions %}
+        <h2 class="ez-list-toolbar-label">{{'locationview.versions.draft.under.edit'|trans({}, 'locationview')}}</h2>
+        {% include '@EzSystemsHybridPlatformUi/content/tabs/versions/table.html.twig' with {'versions': draftVersions} %}
+
+    {% endif %}
+
+    {%  if publishedVersions %}
+        <h2 class="ez-list-toolbar-label">{{'locationview.versions.published.version'|trans({}, 'locationview')}}</h2>
+        {% include '@EzSystemsHybridPlatformUi/content/tabs/versions/table.html.twig' with {'versions': publishedVersions} %}
+    {% endif %}
+
+    {%  if archivedVersions %}
+        <h2 class="ez-list-toolbar-label">{{'locationview.versions.archived.versions'|trans({}, 'locationview')}}</h2>
```

Comment:
I think this translation should be `locationview.versions.archived.version` to keep them consistent. Could this safely be changed in the translations repo?
https://github.com/ezsystems/ezplatform-i18n